.gitignore

```diff
@@ -61,3 +61,4 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
+package-lock.json
```

Comment:
`package-lock.json` [should be committed](https://docs.npmjs.com/cli/v11/configuring-npm/package-lock-json)